README.md

```diff
@@ -93,6 +93,9 @@ merge:
     # to the whitelist.
     pr_body_substrings: ["==MERGE_WHEN_READY=="]
 
+    # Pull requests targeting any of these branches are added to the whitelist.
+    branches: ["develop"]
```

Comment:
this is cool!

non-blocking, for another PR: what's your thoughts on bulldozer providing signal that it will do merge when the checks are ready? i think it may be useful to have comms coming from bulldozer, especially when we start making what is whitelisted more implicit

Reply:
See #70. I think that makes sense, but I'm not sure what mechanism to use to communicate that (status check or something else).